### PR TITLE
Mf/add saved view functionality

### PR DESCRIPTION
**Pull request by mbenfowler**
### Description
This pr will add all new "Saved View" functionality!

### Testing Notes
- When a user clicks the “View Saved Covers” button, we should see the saved covers section, and the homepage view should be hidden
- When the Saved Covers view is visible, the “Show New Random Cover” and “Save Cover” buttons should be hidden
- When the Saved Covers view is visible, the “Home” button should be visible

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -4,7 +4,12 @@ var coverImage = document.querySelector(".cover-image");
...
 var coverDescriptor2 = document.querySelector(".tagline-2");
+var homeButton = document.querySelector('.home-button');
 var showNewRandomCoverButton = document.querySelector('.random-cover-button');
+var saveCoverButton = document.querySelector('.save-cover-button');
+var viewSavedCoversButton = document.querySelector('.view-saved-button');
+var homeView = document.querySelector('.home-view');
+var savedView = document.querySelector('.saved-view');
 
 // We've provided a few variables below
 var savedCovers = [
```
Comment (CaliHam) [inline src/main.js]: We need to see this section!

**Review by CaliHam — CHANGES_REQUESTED** [review]

File: src/main.js [hunk 2]
```diff
@@ -14,6 +19,10 @@ var currentCover;
 
 // Add your event listeners here 👇
 showNewRandomCoverButton.addEventListener('click', getRandomCover);
+viewSavedCoversButton.addEventListener('click', function () {
+  currentView = homeView;
```
Comment (CaliHam) [inline src/main.js:29]: I don't see `currentView` declared anywhere, does it need to be declared first?

File: src/main.js (hunk 1 above)
Comment (mbenfowler) [reply src/main.js]: We actually don't need to see the contents of the `savedCovers` array until iteration3!
But great insight noticing that 😄

File: src/main.js (hunk 2 above)
Comment (mbenfowler) [reply src/main.js:29]: oh shoot, this was the variable that got removed in your last pr but I thought we added it back!  Fantastic catch, I'll add the declaration back!

Comment (mbenfowler) [reply src/main.js:29]: looks like it IS currently on main, so I'll make sure that this line is there as I resolve any conflicts
https://github.com/CaliHam/romcom/blame/main/src/main.js#L19

Comment (CaliHam) [reply src/main.js:29]: That's `var currentCover;` lol 
I just watched a video that kinda describes what's happening here and why it's working tho, interesting stuff! [This video.](https://youtu.be/9emXNzqCKyg?t=469)

File: src/main.js (hunk 1 above)
Comment (CaliHam) [reply src/main.js]: Oh okay, I wasn't sure what was meant by "we should see the saved covers section"; sounds good then!

**Pull request merged**